src/Umbraco.Web.UI.Client/src/common/services/contenteditinghelper.service.js

```diff
@@ -489,6 +489,8 @@ function contentEditingHelper(fileManager, $q, $location, $routeParams, notifica
          *
          * @description
          * A function to handle what happens when we have validation issues from the server side
+         *
+         * TODO: Move to formHelper, so all this is in one place

```

Comment:
@Shazwazza wanting me to action/do something with this now - or is this for a later refactor?